experiments/tester/src/RNTester/TestComponents/Common/styles.ts

```diff
@@ -1,11 +1,116 @@
 import { StyleSheet } from 'react-native';
+import { IStackProps } from '@fluentui-react-native/stack';
 
-export const styles = StyleSheet.create({
+export const commonTestStyles = StyleSheet.create({

```

Comment:
I feel like these should probably use themedStyleSheet like the styles I wrote for the theme test page... it's part of the reason I didn't use sectionHeader like Luan did

Reply:
Alicia, can the styles you wrote be shared in some way? It'd be nice to have a consistent look and feel across all test pages.

Reply:
^ was my intention too, share nicely styled test pages for consistent LNF.  Mind if I push unifying the styles to a new work item?

Reply:
Yea, that's fine.  I almost did it myself but I felt like it was gonna snowball.  No reason we can't share the themed styles I wrote